### PR TITLE
[all hosts] (Manifest, Requirement sets) Moving manifest and req set content to this repo

### PR DESCRIPTION
**Pull request by AlexJerabek**
[Staging site](https://review.docs.microsoft.com/en-us/javascript/api/requirement-sets?view=excel-js-preview&branch=AlexJ-Manifest)

This PR moves the manifest and requirement set information to the reference repo. This is thematically fitting and should help with discovery.

There isn't really net-new content in this PR, but I'm happy to add some to overview pages, if it seems helpful.

### CODE REVIEW

File: docs/docs-ref-autogen/overview/toc.yml [hunk 1]
```diff
@@ -17,3 +17,7 @@ items:
         href: /javascript/api/word
       - name: Common APIs
         href: /javascript/api/office
+  - name: Manifest reference
+    href: /javascript/api/manifest
+  - name: Manifest reference
+    href: /javascript/api/manifest
```
Comment (ElizabethSamuel-MSFT) [inline docs/docs-ref-autogen/overview/toc.yml:23]: Seems like this should be the Requirement sets link?
```suggestion
  - name: Requirement sets
    href: /javascript/api/requirement-sets
```

Comment (AlexJerabek) [reply docs/docs-ref-autogen/overview/toc.yml]: Good catch. I think that proves this file is no longer necessary :-)

**Comment by AlexJerabek** [comment]
@ElizabethSamuel-MSFT, thanks for the review. There seems to be a lingering problem with the Common API moniker fusion TOCs. I'll address that before re-requesting a review.

**Comment by AlexJerabek** [comment]
Closing while I test monikers.

File: docs/requirement-sets/outlook/requirement-set-1.1/outlook-requirement-set-1.1.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+---
+title: Outlook add-in API requirement set 1.1
+description: Features and APIs that were introduced for Outlook Add-ins and the Office JavaScript APIs as part of Mailbox API 1.1.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.1/outlook-requirement-set-1.1.md:3]:
```suggestion
description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.1.
```

File: docs/requirement-sets/outlook/requirement-set-1.2/outlook-requirement-set-1.2.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+title: Outlook add-in API requirement set 1.2
+description: Features and APIs that were introduced for Outlook Add-ins and the Office JavaScript APIs as part of Mailbox API 1.2.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.2/outlook-requirement-set-1.2.md:3]:
```suggestion
description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.2.
```

File: docs/requirement-sets/outlook/requirement-set-1.3/outlook-requirement-set-1.3.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
+---
+title: Outlook add-in API requirement set 1.3
+description: Features and APIs that were introduced for Outlook Add-ins and the Office JavaScript APIs as part of Mailbox API 1.3.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.3/outlook-requirement-set-1.3.md:3]:
```suggestion
description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.3.
```

File: docs/requirement-sets/outlook/requirement-set-1.4/outlook-requirement-set-1.4.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
+---
+title: Outlook add-in API requirement set 1.4
+description: Features and APIs that were introduced for Outlook Add-ins and the Office JavaScript APIs as part of Mailbox API 1.4.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.4/outlook-requirement-set-1.4.md:3]:
```suggestion
description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.4.
```

File: docs/requirement-sets/outlook/requirement-set-1.5/outlook-requirement-set-1.5.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
+---
+title: Outlook add-in API requirement set 1.5
+description: Features and APIs that were introduced for Outlook Add-ins and the Office JavaScript APIs as part of Mailbox API 1.5.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.5/outlook-requirement-set-1.5.md:3]:
```suggestion
description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.5.
```

File: docs/requirement-sets/outlook/requirement-set-1.6/outlook-requirement-set-1.6.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
+---
+title: Outlook add-in API requirement set 1.6
+description: Features and APIs that were introduced for Outlook Add-ins and the Office JavaScript APIs as part of Mailbox API 1.6.
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.6/outlook-requirement-set-1.6.md:3]:
```suggestion
description: Features and APIs that were introduced for Outlook add-ins and the Office JavaScript APIs as part of Mailbox API 1.6.
```

File: docs/requirement-sets/outlook/requirement-set-1.7/outlook-requirement-set-1.7.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
+---
+title: Outlook add-in API requirement set 1.7
+description: Overview of the Outlook add-in API (requirement set 1.7)
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/outlook/requirement-set-1.7/outlook-requirement-set-1.7.md:3]:
```suggestion
description: Overview of the Outlook add-in API (requirement set 1.7).
```

File: docs/requirement-sets/powerpoint/powerpoint-api-requirement-sets.md [hunk 9]
```diff
@@ -0,0 +1,81 @@
...
+<Requirements>
+   <Sets DefaultMinVersion="1.1">
+      <Set Name="PowerPointApi" MinVersion="1.1"/>
+   </Sets>
+</Requirements>
+```
+
+## Office Common API requirement sets
+
+Most of the PowerPoint Add-in functionality comes from the Common API set. For information about Common API requirement sets, see [Office Common API requirement sets](../common/office-add-in-requirement-sets.md).
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/powerpoint/powerpoint-api-requirement-sets.md:74]:
```suggestion
Most of the PowerPoint add-in functionality comes from the Common API set. For information about Common API requirement sets, see [Office Common API requirement sets](../common/office-add-in-requirement-sets.md).
```

File: docs/requirement-sets/word/word-api-1-1-requirement-set.md [hunk 10]
```diff
@@ -0,0 +1,22 @@
+---
+title: Word JavaScript API requirement set 1.1
+description: Details about the WordApi 1.1 requirement set
```
Comment (ElizabethSamuel-MSFT) [inline docs/requirement-sets/word/word-api-1-1-requirement-set.md:3]:
```suggestion
description: Details about the WordApi 1.1 requirement set.
```

**Review by ElizabethSamuel-MSFT — COMMENTED** [review]
Making sure to capture other changes to this content from the metadata PR

**Comment by AlexJerabek** [comment]
Thanks @ElizabethSamuel-MSFT. I accepted your changes and double-checked against the /reference files in the metadata description PR. Should be good to go now.

**Review by ElizabethSamuel-MSFT — APPROVED** [review]

**Pull request merged**